src/parser/jobs/brd/modules/RagingStrikes.tsx

```diff
@@ -1,25 +1,37 @@
 import {t} from '@lingui/macro'
-import {Trans} from '@lingui/react'
-import {ActionLink, StatusLink} from 'components/ui/DbLink'
+import {Plural, Trans} from '@lingui/react'
+import {ActionLink, DataLink, StatusLink} from 'components/ui/DbLink'
```

Comment:
Since you're using `DataLink` here we could just switch over the existing Links to use it instead of importing all three.